jobs/rocoto/prep.sh

```diff
@@ -1,129 +1,141 @@
-        if [ ! -s $ARCHSYND/syndat_akavit ]; then
+        if [[ ! -d ${ARCHSYND} ]]; then mkdir -p ${ARCHSYND}; fi
+        if [[ ! -s ${ARCHSYND}/syndat_akavit ]]; then
             for file in syndat_akavit syndat_dateck syndat_stmcat.scr syndat_stmcat syndat_sthisto syndat_sthista ; do
-                cp $COMINsyn/$file $ARCHSYND/.
+                cp ${COMINsyn}/${file} ${ARCHSYND}/.
             done
         fi
     fi
 
-    [[ $ROTDIR_DUMP = "YES" ]] && rm $COMOUT${CDUMP}.t${cyc}z.syndata.tcvitals.tm00
+    [[ ${ROTDIR_DUMP} = "YES" ]] && rm ${COMOUT}${CDUMP}.t${cyc}z.syndata.tcvitals.tm00
 
-    $HOMEgfs/jobs/JGLOBAL_ATMOS_TROPCY_QC_RELOC
+    ${HOMEgfs}/jobs/JGLOBAL_ATMOS_TROPCY_QC_RELOC
     status=$?
-    [[ $status -ne 0 ]] && exit $status
+    [[ ${status} -ne 0 ]] && exit ${status}
 
 else
-    [[ $ROTDIR_DUMP = "NO" ]] && cp $DMPDIR/${CDUMP}${DUMP_SUFFIX}.${PDY}/${cyc}/${COMPONENT}/${CDUMP}.t${cyc}z.syndata.tcvitals.tm00 $COMOUT/
+    [[ ${ROTDIR_DUMP} = "NO" ]] && cp ${DMPDIR}/${CDUMP}${DUMP_SUFFIX}.${PDY}/${cyc}/${COMPONENT}/${CDUMP}.t${cyc}z.syndata.tcvitals.tm00 ${COMOUT}/
 fi
 
+# Will modify the new location later in the next PR to address issue 1198
+if [[ ${ROTDIR_DUMP} = "YES" ]]; then
+   mv ${COMIN_OBS}/*syndata.tcvitals.tm00 ${COMOUT}
+   mv ${COMIN_OBS}/*snogrb_t1534.3072.1536 ${COMOUT}
+   mv ${COMIN_OBS}/*seaice.5min.blend.grb ${COMOUT}
+fi
 
 ###############################################################
 # Generate prepbufr files from dumps or copy from OPS
-if [ $MAKE_PREPBUFR = "YES" ]; then
-    if [ $ROTDIR_DUMP = "YES" ]; then
-        rm -f $COMOUT/${OPREFIX}prepbufr
-        rm -f $COMOUT/${OPREFIX}prepbufr.acft_profiles
-        rm -f $COMOUT/${OPREFIX}nsstbufr
+if [[ ${MAKE_PREPBUFR} = "YES" ]]; then
+    if [[ ${ROTDIR_DUMP} = "YES" ]]; then
+        rm -f ${COMIN_OBS}/${OPREFIX}prepbufr
+        rm -f ${COMIN_OBS}/${OPREFIX}prepbufr.acft_profiles
+        rm -f ${COMIN_OBS}/${OPREFIX}nsstbufr
     fi
 
     export job="j${CDUMP}_prep_${cyc}"
-    export DATAROOT="$RUNDIR/$CDATE/$CDUMP/prepbufr"
+    export DATAROOT="${RUNDIR}/${CDATE}/${CDUMP}/prepbufr"
     #export COMIN=${COMIN:-$ROTDIR/$CDUMP.$PDY/$cyc/$COMPONENT}
-    export COMIN=${COMIN:-$ROTDIR}
-    export COMINgdas=${COMINgdas:-$ROTDIR/gdas.$PDY/$cyc/$COMPONENT}
-    export COMINgfs=${COMINgfs:-$ROTDIR/gfs.$PDY/$cyc/$COMPONENT}
-    if [ $ROTDIR_DUMP = "NO" ]; then
-        COMIN_OBS=${COMIN_OBS:-$DMPDIR/${CDUMP}${DUMP_SUFFIX}.${PDY}/${cyc}/${COMPONENT}}
-        export COMSP=${COMSP:-$COMIN_OBS/$CDUMP.t${cyc}z.}
+    #export COMIN=${COMIN:-$ROTDIR}
+    export COMIN=${COMIN_OBS}
+    export COMINgdas=${COMINgdas:-${ROTDIR}/gdas.${PDY}/${cyc}/${COMPONENT}}
+    export COMINgfs=${COMINgfs:-${ROTDIR}/gfs.${PDY}/${cyc}/${COMPONENT}}
+    export COMOUT=${COMIN_OBS}
+    if [[ ${ROTDIR_DUMP} = "NO" ]]; then
+        COMIN_OBS=${COMIN_OBS:-${DMPDIR}/${CDUMP}${DUMP_SUFFIX}.${PDY}/${cyc}/${COMPONENT}}
```

Comment:
```suggestion
        COMIN_OBS=${COMIN_OBS:-${DMPDIR}/${CDUMP}${DUMP_SUFFIX}.${PDY}/${cyc}/atmos}
```
`DMPDIR` has `atmos` hard-wired.